ocs_ci/ocs/bucket_utils.py

```diff
@@ -937,7 +939,11 @@ def s3_put_object(s3_obj, bucketname, object_key, data, content_type=""):
 
     """
     return s3_obj.s3_client.put_object(
```

Comment:
Verify whether this change does not cause any failures of tests using  s3_obj.s3_client.put_object(

Reply:
ack